app/code/Magento/InventoryInStorePickupShippingApi/Model/Carrier/GetCarrierTitle.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupShippingApi\Model\Carrier;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+
+/**
+ * Get In-Store Pickup carrier title
+ *
+ * @api
+ */
+class GetCarrierTitle
```

Comment:
Can we use this service in \Magento\InventoryInStorePickupShippingApi\Model\Carrier\InStorePickup::createResultMethod
To make consistent receiving of carrier title?

Reply:
ok